lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+path = require 'path'
+
+module.exports =
+  config:
+    executable:
+      type: 'string'
+      default: 'pylint'
+    rcFile:
```

Comment:
I think it can be useful to be able to use a rc file that lives in a different location